packages/reports/tests/acceptance/report-test.js

```diff
@@ -76,12 +79,18 @@ module('Acceptance | Navi Report', function(hooks) {
   test('Clone invalid report', async function(assert) {
     assert.expect(1);
 
-    await visit('/reports/1');
-    // Add a metric filter
-    await clickItemFilter('dimension', 'Operating System');
+    await visit('/reports/13');
+    // Add a dimension filter
+    await click('.navi-column-config-item__remove-icon[aria-label="delete metric Ad Clicks"]');
+    await click('.navi-column-config-item__remove-icon[aria-label="delete dimension Property (id)"]');
+    await click('.navi-column-config-item__remove-icon[aria-label="delete time-dimension Date Time (day)"]');
     await click($('.navi-report__action-link:contains(Clone)')[0]);
 
-    assert.ok(currentURL().endsWith('edit'), 'An invalid new report transitions to the reports/:id/edit route');
+    //fails because of server error and stays at /reports/13/edit
+    assert.ok(
+      currentURL().endsWith('/edit'),
+      'After clicking the run button, the route transitions to the invalid route'
+    );
```

Comment:
This should move the user over to a new report, not stay at `/13/edit`, can we assert that it moves the user to a new report. Maybe assert the url does not include `/reports/13`?

Reply:
It moves to a new report edit, is that whats intended?

Reply:
Yea, the comment `//fails because of server error and stays at /reports/13/edit` indicated otherwise

Reply:
We should remove the comment^ and add an assert that the url does not contain `/reports/13`